pkg/registry/core/service/ipallocator/metrics.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ipallocator
+
+import (
+	"sync"
+
+	"k8s.io/component-base/metrics"
+	"k8s.io/component-base/metrics/legacyregistry"
+)
+
+const (
+	namespace = "kube_apiserver"
+	subsystem = "clusterip_allocator"
+)
+
+var (
+	// clusterIPAllocated indicates the amount of cluster IP allocated by Service CIDR.
+	clusterIPAllocated = metrics.NewGaugeVec(
```

Comment:
why are these not part of `NewAllocatorCIDRRange()`

Reply:
they are https://github.com/kubernetes/kubernetes/pull/104119/files#diff-3f7680212eb37166b66a0f052153bb765c0d4b141af958d5b92543dda628d35fR194